internal/instancehealth/checks.go

```diff
@@ -134,22 +134,24 @@ func checkPermissionsSyncing(
 	var syncCount int
 	var syncErrors []string
 	var seenProviders = make(map[string]map[string]string) // provider : state : message
-	for _, sync := range instanceHealth.PermissionsSyncJobs.Nodes {
-		if sync.CompletedAt.Before(time.Now().Add(-since)) {
+	for _, sync := range instanceHealth.PermissionSyncJobs.Nodes {
+		if sync.FinishedAt.Before(time.Now().Add(-since)) {
 			continue
 		}
 		syncCount += 1
-		if sync.Status == "ERROR" {
-			syncErrors = append(syncErrors, sync.Message)
+		if sync.State == "ERROR" {
```

Comment:
sync.State == "ERROR" || sync.State == "FAILURE"

Reply:
it was not the part of original logic, but I agree.

@bobheadxi ping, just in case :)

Reply:
+1, the `FAILURE` state is new IIRC so let's include it :)